packages/react-components/src/components/CameraButton.tsx

```diff
@@ -240,6 +228,18 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     });
   }
 
+  splitButtonMenuItems.push({
+    key: 'cameraPrimaryAction',
+    text: props.checked ? strings.onSplitButtonPrimaryActionCamera : strings.offSplitButtonPrimaryActionCamera,
+    onClick: () => {
+      onToggleClick();
+    },
+    iconProps: {
+      iconName: props.checked ? 'SplitButtonPrimaryActionCameraOn' : 'SplitButtonPrimaryActionCameraOff',
+      styles: { root: { lineHeight: 0 } }
```

Comment:
I wonder if based on Alex's new designs we should be stripping the blue colour off the icons and using a new semantic colour from the theme? This would go for all the icons in the control bar

Reply:
I verified with Alex and he says the current one is fine